packages/webpack/src/buildtime/wrapper.js

```diff
@@ -33,13 +38,26 @@ exports.wrapper = function wrapper({
   runtimeKit,
   evalKitFunctionName,
   runChecks = true,
+  runtimeFlags = {},
 }) {
+  const runtimeKitArray = Array.from(runtimeKit)
   // validateSource(source);
 
   // No AST used in these transforms, so string cmparison should indicate if anything was changed.
   const sesCompatibleSource = applySourceTransforms(source)
   const sourceChanged = source !== sesCompatibleSource
 
+  // This adds support for mapping `this` to `exports` or `module.exports` if webpack detected it's necessary
+  let optionalBinding = ''
+
+  if (runtimeFlags.thisAsExports) {
+    if (runtimeKitArray.includes('exports')) {
+      optionalBinding = '.bind(exports)'
+    } else if (runtimeKitArray.includes('module')) {
+      optionalBinding = '.bind(module.exports)'
+    }
```

Comment:
in case it has any bearing on this logic: you can use both `exports` and `module.exports` in the same damn script if you want

Reply:
yes, this is here because if 'exports' are not used, webpack doesn't list them as a requirement and they're actually undefined in the scope of the module, you have to reach for `module.exports` to grab that reference.